guides/get-going-with-gitops.mdx

```diff
@@ -0,0 +1,485 @@
+---
+title: Get going with GitOps
+description: Configuring Flipt for a GitOps workflow
+---
+
+## Why GitOps?
+
+GitOps is a set of practices centered around storing your application and infrastructure configuration in Git.
+The goal being to leverage the capabilities of version-control for your entire systems configuration.
+
+If you're already embracing GitOps practices for your other configuration, then aligning your feature flags with these same practices can complete the experience.
+No more correlating what changed in Git, with what changed in your feature flag system to understand the entire state of the world.
+Git is the single source of truth as GitOps intended.
+
+So how do we achieve this with Flipt?
+
+## What We Will Learn
+
+In this guide we will:
+
+- 🏁 Add a feature flag to an existing codebase
+- 📝 Define the flag via Flipt's configuration format
+- 🌲 Add, commit and push the change to a production serving branch
+- 🎯 Adjust our configuration to target an internal group of users in our organization
+- 🌓 Progressively enable the flag for proportions of the user base
+
+## Setting the Scene
```

Comment:
Maybe from here on down it makes sense to use `we`? like you already are doing?

Reply:
Yeah I think that makes sense.